app/controllers/restrooms_controller.rb

```diff
@@ -62,7 +57,25 @@ def destroy
     end
   end
 
+  def upvote
```

Comment:
The issue with moving these to custom actions is then we are not following strict REST / CRUD. Which has advantages and disadvantages. As a note. The original code I wrote had these as custom actions, but it was suggested by many that we stick to a strict CRUD since these are just attributes on the table there is no need to have custom actions. 

However, I realize with papertrail it may not be possible to accept updating these columns without making them custom actions. In this case. Is it possible to use just one endpoint /vote or something? This endpoint could taken in a parameter for upvote / downvote ?